packages/ui/components/Pages/Index/TableItem.tsx

```diff
@@ -0,0 +1,61 @@
+export type TableDataItems = {
+  name: string;
+  status: string;
+  owner: string;
+  budget: string;
+  duration: string;
+  startDate: string;
+  endDate: string;
+};
+type TableItemProps = {
+  item: TableDataItems;
+};
+
+const TableItem = ({ item }: TableItemProps) => {
+  return (
+    <tr
+      key={item.name}
+      className="text--gray text-sm rc-border-1-gray-100"
+    >
+      <td className="rc-border-1-gray-100 p-2">{item.name}</td>
+      <td className="rc-border-1-gray-100 text-white p-2">
+        <div className={`rc-status-${item.status}`}>
+          {item.status}
+        </div>
+      </td>
+      {item.owner !== '' ? (
```

Comment:
Next step is to move these into their own components. They will have an interaction like a dropdown etc